### PR TITLE
Dogecoin support -- wow

### LINKED ISSUE #2685: support dogecoin
Reticketing from #2048 to avoid spamming people.

<bountysource-plugin>
---
There is a **[$15 open bounty](https://www.bountysource.com/issues/4040573-support-dogecoin?utm_campaign=plugin&utm_content=tracker%2F85909&utm_medium=issues&utm_source=github)** on this issue. Add to the bounty at [Bountysource](https://www.bountysource.com/?utm_campaign=plugin&utm_content=tracker%2F85909&utm_medium=issues&utm_source=github).
</bountysource-plugin>

### PR DESCRIPTION
**Pull request by rht**
Someone needs to do it. resolves #2685.

### CODE REVIEW

File: www/%username/dogecoin.json.spt [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+from gratipay.utils import bitcoin
+
+[---------------------------------------------------------------]
+
+if user.ANON:
+    raise Response(404)
+request.allow("POST")
+
+bit_address = request.body['dogecoin_address'].strip()
+if bit_address != '' and not bitcoin.validate(bit_address, version=0x1E):
```
Comment (seanlinsley) [inline www/%username/dogecoin.json.spt]: What does this version parameter do?

Comment (rht) [inline www/%username/dogecoin.json.spt]: ```version=0``` is for bitcoin, ```version=0x1E``` is for dogecoin.

**Comment by Changaco** [comment]
This is no longer blocked, changes to facilitate adding other cryptocoins have been made in #3282.

**Comment by whit537** [comment]
Changed to `WIP`, because `Ready to Start` doesn't make sense for PRs&mdash;the work is already started! If we don't want to use any of the work already on this PR then let's close this and mark #2685 `Ready to Start`.

**Comment by rht** [comment]
But coinbase doesn't accept dogecoin yet.

**Comment by rohitpaulk** [comment]
> But coinbase doesn't accept dogecoin yet.

We don't have to use coinbase to display a dogecoin address on one's profile.

**Comment by benhc123** [comment]
I think Dogecoin has gone out of fashion now. We need support for Bitcoin but for others it's less vital.

**Comment by whit537** [comment]
... and I'm not sure we need Bitcoin either (https://github.com/gratipay/inside.gratipay.com/issues/201#issuecomment-102010517), but that's a separate issue.

**Pull request closed without merging**